dev-docs/bidders/redtram.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bidder
+title: Redtram
+description: Prebid Redtram Bidder Adapter.
+pbjs: true
+biddercode: redtram
+media_types: banner
+---
+
+### Table of Contents
+
+- [Description](#redtram-bid-desc)
+- [Bid Params](#redtram-bid-params)
+- [Test Params](#redtram-test-params)
+
+<a name="redtram-bid-desc" />
+
+### Description
+
+Redtram header bidding adapter connects with redtram demand sources to fetch bids for display placements. Please reach out to your account manager or <support@redtram.com> for more information.
+
+<a name="redtram-bid-params" />
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+| Name         | Scope    | Description                        | Example    | Type     |
+|--------------|----------|------------------------------------|------------|----------|
+| `placementId`| optional | The placement ID from Redtram      | `'23611'`  | `string` |
+| `bidFloor`   | optional | Lowest value of expected bid price | `1.1`      | `float`  |
+| `currency`   | optional | Currency of request and response   | `'USD'`    | `string` |
```

Comment:
I don't think you should be accepting currency as a parameter. Instead, you should be reading the currency set by the publisher using the currency module. 
```
config.getConfig('currency.adServerCurrency')
```

Reply:
Removed this parameter from the description.